README.md

```diff
@@ -43,6 +43,8 @@ To protect this Lambda from other rouge Lambdas consuming all the available shar
 
 This value should be set to at least the number of Kinesis trigger shards.
 
+## Deployment
 
+This app is currently only deployed to PROD: there is no CODE stack. Also, [the Riff Raff configuration](./riff-raff.yaml) is only set to deploy the lambda updates: it doesn’t update the cloudformation.
 
-
+There is also no continuous deployment configured in Riff Raff for this app at the moment: since one must manually update the cloudformation, it’s probably safest to let people deploy when they’re ready, so that they can make sure to apply [the cloudformation](./cloudformation.yaml) first.
```

Comment:
Might be worth prefixing this with something like "If your changes are to the cloudformation, this will need to be updated manually prior to deployment, and there is no continuous deployment configured..." etc.

Reply:
Yep, we do that for [Concierge](https://github.com/guardian/content-api/blob/f61e8cc043b75589533d6cfb67619450b9fcc3f6/concierge/riff-raff.yaml#L6-L10) and others.

Reply:
Obviously that ☝️  is in the wrong comment position 🤣 !

Reply:
Changes made!